src/onnx/parse_matmulnbits.cpp

```diff
@@ -67,7 +67,8 @@ struct parse_matmulnbits : op_parser<parse_matmulnbits>
                            ". Actual dims: " + to_string_range(args[1]->get_shape().lens()));
 
         std::vector<size_t> expected_scales_lens{n * n_blocks_per_col};
```

Comment:
I dont think we need to make a vector to store an single integer in this case.

Reply:
Removed